docs/sources/tempo/troubleshooting/alloy.md

```diff
@@ -33,6 +34,24 @@ exporter_sent_spans_ratio_total
 exporter_send_failed_spans_ratio_total
 ```
 
+If you are using Grafana Alloy to send traces to Grafana Cloud, the metrics are visible at
+`http://localhost:12345/metrics`.
```

Comment:
This is a bit misleading, this endpoint is local and it states Grafana Cloud

Reply:
This is referring to the metrics emitted by Alloy itself, `/metrics` is its prometheus scrape endpoint. These should also end up in Grafana Cloud, but in case remote write isn't working yet this is a quick and easy way to check the metrics locally.

I think the bit that might be confusing is _If you are using Grafana Alloy_ since this page is about Alloy already.

Reply:
Good points. I"ll write a suggestion to clarifiy.